src/AddTodoForm.js

```diff
@@ -2,12 +2,19 @@ import React from "react";
 
 
 // Function to add form for todo list
-function AddTodoForm(){
+function AddTodoForm(props){
+    const handleAddTodo  = (event) => {
+        event.preventDefault();
+        const todoTitle = event.target.title.value;
+        console.log(todoTitle);
+        event.target.reset();
+        props.onAddTodo(todoTitle);
```

Comment:
don't forget to set the field blank after submission. I recommend `event.target.title.value = ""`.

Reply:
is it not good to use event.target.reset(); to blank all fields from the form